plash_cli/core.py

```diff
@@ -136,27 +136,29 @@ def _tarz(path)->io.BytesIO: # Buffer of tar directory
     tarz.seek(0)
     return tarz, len(files)
 
-# %% ../nbs/00_core.ipynb 18
+# %% ../nbs/00_core.ipynb 17
 @call_parse
 def deploy(
     path:Path=Path('.'), # Path to project
     local:bool=False,    # Local dev
-    port:int=5002):      # Port for local dev
+    port:int=5002,       # Port for local dev
+    app_id:str=None):    # App ID that will be used as the subdomain in plash
+
     """🚀 Ship your app to production"""
     print('Initializing deployment...')
     # Check if path is a file
     if path.is_file():
-        tarz, _ = _tarz(path.read_bytes())
-        plash_app = path.parent / '.plash'
+        tarz, _ = _tarz(path)
```

Comment:
I think we can make this more concise. I see this line occurs both in the if and else part.